src/languages/lua.coffee

```diff
@@ -19,4 +19,9 @@ module.exports = {
 
   defaultBeautifier: "Lua beautifier"
 
+  options:
+    do_not_change_whitespace:
```

Comment:
I do not like the name of this option. Is there a clearer way to display this to the users?

Do we even need it? I think what `adjust_space` is doing is adding/removing whitespace. Is that not something we want the beautifier to always do?